src/pages/PingDetail.svelte

```diff
@@ -65,7 +70,11 @@
     title={"Access"}
     helpText={"Ways to access this metric in Mozilla's data warehouse."}
   />
-
+  {#if isRecent(ping)}
+    <AppAlert
+      status="warning"
```

Comment:
I think it would be nice to have an "info" app alert with this type of information, since the warning one is a little alarming. Definitely something to do in a followup though.